core-bundle/src/Resources/contao/languages/en/tl_page.xlf

```diff
@@ -116,6 +116,12 @@
       <trans-unit id="tl_page.validAliasCharacters.1">
         <source>Here you can select a custom character set for auto-generated aliases.</source>
       </trans-unit>
+      <trans-unit id="tl_page.useFolderUrl.0">
+        <source>Enable folder URLs</source>
+      </trans-unit>
+      <trans-unit id="tl_page.useFolderUrl.1">
+        <source>Here you can enable folder-style page aliases like &lt;em&gt;docs/install/download.html&lt;/em&gt; instead of &lt;em&gt;docs-install-download.html&lt;/em&gt;.</source>
```

Comment:
We should adjust the text so the user knows that enabling the checkbox will **generate** folder-style page aliases and not **enable** them (because now they are enabled all the time).